src/components/OSCALCatalogGroups.js

```diff
@@ -52,32 +53,64 @@ const ComponentTab = styled(Tab)(({ theme }) => ({
   textTransform: "none",
 }));
 
-function a11yProps(index, title) {
+function a11yProps(groupId) {
   return {
-    id: `vertical-tab-${index}`,
-    "aria-controls": `vertical-tabpanel-${title}`,
+    id: `vertical-tab-${groupId}`,
+    "aria-controls": `vertical-tab-${groupId}`,
   };
 }
 
 TabPanel.propTypes = {
-  index: PropTypes.number.isRequired,
-  value: PropTypes.number.isRequired,
+  value: PropTypes.string.isRequired,
 };
 
 export default function OSCALCatalogGroups(props) {
-  const [value, setValue] = React.useState(0);
+  const [value, setValue] = React.useState(props?.groups[0]?.id);
```

Comment:
There must be a better name for this than `value`? If we're refactoring a bunch of this anyway...

Reply:
I totally agree. When looking tracing the code at first, I was confused what `value` was referring to. From my understanding this is the current control grouping tab that is open. Perhaps this is renamed to something like "openControlGroupingTab" or simply "openTab".